### PR TITLE
ingest embargoed in newsml g2 and publish it in ninjs

### PR DESCRIPTION
**Pull request by petrjasek**
STTNHUB-56

### CODE REVIEW

File: superdesk/publish/formatters/ninjs_formatter.py [hunk 1]
```diff
@@ -173,7 +173,13 @@ def _transform_to_ninjs(self, article, subscriber, recursive=True):
         if extra_items:
             ninjs.setdefault(EXTRA_ITEMS, {}).update(extra_items)
 
-        if article.get(EMBARGO):
+        if article.get('embargoed'):
+            try:
+                ninjs['embargoed'] = article['embargoed'].isoformat()
+            except AttributeError:
```
Comment (jerome-poisson) [inline superdesk/publish/formatters/ninjs_formatter.py]: In which case the `isoformat` attribute may be missing? If it's a datetime, should not be always present?

Comment (petrjasek) [reply superdesk/publish/formatters/ninjs_formatter.py]: just in case when it gets ingested and it might not parse the format properly

Comment (jerome-poisson) [reply superdesk/publish/formatters/ninjs_formatter.py]: @petrjasek in this case maybe something should notify the user, a `logger.warning` for instance? That would indicate that something is wrong with either the file format or the parser.

Comment (petrjasek) [reply superdesk/publish/formatters/ninjs_formatter.py]: good point, will update

**Pull request merged**